client_environment.yml

```diff
@@ -19,7 +20,10 @@ dependencies:
   - notebook
   # for pip packages
   - pip
+  - nc-time-axis
   - pip:
     # visual debugger for Jupyter Notebook, not working with JupyterLab at this moment
+    - owslib<0.22
     - pixiedust
     - birdhouse-birdy
+    - ravenpy
```

Comment:
FYI, installing RavenPy like this won't have the binaries, is that okay?  I am not sure what `client_environment.yml` is used for so maybe it does not require the binaries so it's fine.

Reply:
Yes, I think this is used only to get the test files.